dashboard/lib/services/lti.rb

```diff
@@ -4,231 +4,235 @@
 require 'authentication_option'
 require 'sections/section'
 
-class Services::Lti
-  def self.initialize_lti_user(id_token)
-    user_type = Policies::Lti.get_account_type(id_token[Policies::Lti::LTI_ROLES_KEY])
-    user = User.new
-    user.provider = User::PROVIDER_MIGRATED
-    user.user_type = user_type
-    if user_type == User::TYPE_TEACHER
-      user.age = '21+'
-      user.name = get_claim_from_list(id_token, Policies::Lti::TEACHER_NAME_KEYS)
-      user.lti_roster_sync_enabled = true
-    else
-      user.name = get_claim_from_list(id_token, Policies::Lti::STUDENT_NAME_KEYS)
-      user.family_name = get_claim(id_token, :family_name)
```

Comment:
Refactored `Services::Lti` into a module.
It makes more sense for this service to be a module since we are not creating any instance of `Service::Lti` anywhere in the code.

Reply:
Does it make sense to move this into the lti directory as well?

Reply:
potentially, but many more changes will be needed to further refactor `Servce::Lti`. I wanted to keep the changes related to the bug as much as possible.

Reply:
We do probably want to refactor it eventually, but would moving it break anything? Does the location of the file affect how it's referenced?

Reply:
In theory, Rails autoloads all modules, so a location change should not affect it.
We could also specify the new `require` path: `require 'services/lti/some_lti_service'`.
We would have to update all the calls from `Services::Lti` to `Services::Lti::SomeLtiService`. this is the reason I did not want to add more changes to this PR.

Reply:
We don't need to (and defenelty don't have to) move all of the methods.
But I'd suggest moving those that are complex enough to benefit from encapsulation into their own class.

Reply:
We could transform `Services::Lti` into a [Facade](https://refactoring.guru/design-patterns/facade). This way, it would serve as the interface for LTI-related functionality, while the business logic would be split among separate service objects:
```ruby
module Services
  module Lti
    def self.initialize_lti_user(id_token)
      Services::Lti::UserInitializer.new(id_token).call
    end
    
    def self.create_lti_user_identity(user)
      Services::Lti::UserIdentetyCreator.new(user).call
    end
    
    ...
  end
end
```